contrib/libunwind-cmake/CMakeLists.txt

```diff
@@ -43,7 +43,10 @@ set_target_properties(unwind PROPERTIES FOLDER "contrib/libunwind-cmake")
 
 target_include_directories(unwind SYSTEM BEFORE PUBLIC $<BUILD_INTERFACE:${LIBUNWIND_SOURCE_DIR}/include>)
 target_compile_definitions(unwind PRIVATE -D_LIBUNWIND_NO_HEAP=1 -D_DEBUG -D_LIBUNWIND_IS_NATIVE_ONLY)
-target_compile_options(unwind PRIVATE -fno-exceptions -funwind-tables -fno-sanitize=all $<$<COMPILE_LANGUAGE:CXX>:-nostdinc++ -fno-rtti>)
+
+# We should enable optimizations (otherwise it will be too slow in debug)
+# and disable sanitizers (otherwise infinite loop may happen)
+target_compile_options(unwind PRIVATE -O3 -fno-exceptions -funwind-tables -fno-sanitize=all $<$<COMPILE_LANGUAGE:CXX>:-nostdinc++ -fno-rtti>)
```

Comment:
Maybe you can also switch to `-O2` instead here and here https://github.com/ClickHouse/ClickHouse/blob/d8924f0b0ed96c5b6bc4398f34aa7cc96dcaf8d2/src/CMakeLists.txt#L342-L347

This will allow to debug unwind issues in debug builds w/o rebuilding.

Reply:
Ok.